DigitalOffice/Tests/FileServiceTests/Validators/AddNewFileValidatorTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using FileService.RestRequests;
+using FileService.Validators;
+using FluentValidation;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+
+namespace FileServiceUnitTests.Validators
+{
+    public class AddNewFileRequestValidatorTests
+    {
+        private IValidator<FileRequest> validator;
+        private FileRequest fileRequest;
+
+        [SetUp]
+        public void Initialization()
+        {
+            fileRequest = new FileRequest
+            {
+                Content = "RGlnaXRhbCBPZmA5Y2U=",
+                Extension = ".txt",
+                Name = "DigitalOfficeTestFile"
+            };
+
+            validator = new NewFileValidator();
+        }
+
+        [Test]
+        public void SuccessfulFileValidationTest()
+        {
+            var fileValidationResult = validator.TestValidate(fileRequest);
+            fileValidationResult.ShouldNotHaveAnyValidationErrors();
```

Comment:
wouldn't it be easier here and further in the validator tests to make it like:
`validator.ShouldNotHaveValidationErrorFor(request)`

Reply:
good idea

Reply:
@Inzilbeth but I cant't figure out how to pass LINQ as the first argument. 
So i used result variable and test it for no errors at all. So i leave it like that for the time being.

link from Slack: https://docs.fluentvalidation.net/en/latest/testing.html